core/templates/core/index.html

```diff
@@ -3,7 +3,9 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
-    <div class="flex items-center justify-center h-screen text-white">
+    <canvas id="starback-bg" class="fixed -z-1"></canvas>
+
+    <div class="flex items-center justify-center h-screen">
```

Comment:
Add relative z-1 to this div, so it displays above background

Reply:
What?

Reply:
You have position: relative and something-z: 1 in css, and tailwind has classes for it, for position: relative `relative` class and for something-z: 1 `z-1` class, add them to this div

Reply:
And you assume i understand all that?

Reply:
You have to